CustomEd.Classroom.Service/appsettings.json

```diff
@@ -18,7 +18,7 @@
     "TokenLifetime": 60
   },
   "RabbitMQSettings":{
-    "Host":"localhost"
+    "Host":"amqps://xrbirzby:N2XAE5oEzUjXETriGu4rsxWlzhBuEfxz@kangaroo.rmq.cloudamqp.com/xrbirzby"
```

Comment:
this should not be pushed. Add them to .gitignore

Reply:
Sure I'll.